pkg/store/bucket_test.go

```diff
@@ -269,3 +273,24 @@ func TestPartitionRanges(t *testing.T) {
 		testutil.Equals(t, c.expected, res)
 	}
 }
+
+func TestBucketStore_Info(t *testing.T) {
+	defer leaktest.CheckTimeout(t, 10*time.Second)()
+
+	ctx, cancel := context.WithCancel(context.Background())
+	defer cancel()
+
+	dir, err := ioutil.TempDir("", "prometheus-test")
+	testutil.Ok(t, err)
+
+	// TODO: might be worth creating a `testutil.NewBucketStore()`
```

Comment:
I haven't actually executed this code but LGTM. Except for this comment - what do you mean by this?

Reply:
A bit like https://github.com/improbable-eng/thanos/blob/00619585c5dc3d0ce2275a8864e3b666d8375acc/pkg/testutil/prometheus.go#L76 to create a test instance.

Reply:
Well the only benefit would be to use shorter function instead of (single line anyway) ` NewBucketStore(nil, nil, nil, dir, 2e5, 2e5, false, 20)`.

Let's remove this comment and go with what it is here :+1: